ferrocene/doc/sphinx-shared-resources/requirements.in

```diff
@@ -2,7 +2,9 @@
 # SPDX-FileCopyrightText: The Ferrocene Developers
 
 sphinx
+sphinx-needs
 sphinx-autobuild
+sphinxcontrib-plantuml
```

Comment:
You need to setup the extension only if you want UML graphs to be generated, which right now we don't need.

Reply:
So you added it for the case that we want UML diagrams? Which we do at some point?

Reply:
So, the dependency is added to `requirements.in` / `requirements.txt` because if the Python package is not installed `sphinx-needs` will crash at startup. *Enabling* the extension for the Sphinx project is optional, and I don't see us needing that.